packages/expo-module-scripts/package.json

```diff
@@ -27,10 +27,17 @@
   "homepage": "https://github.com/expo/expo/tree/main/packages/expo-module-scripts#readme",
   "dependencies": {
     "@babel/cli": "^7.1.2",
+    "@babel/plugin-proposal-class-properties": "^7.18.6",
+    "@babel/plugin-proposal-export-namespace-from": "^7.18.9",
```

Comment:
Would like to eventually drop this in SDK 50 or 51. The proposal has been stalled for years and the amount of convenience it adds is minimal.

Reply:
Unfortunately, it's a [requirement for Reanimated](https://github.com/software-mansion/react-native-reanimated/issues/3364). Granted, on the server we likely shouldn't be using Reanimated, but IMHO, with SSG being supported in expo-router, we might need to keep it.

PS. it's also included in `babel-preset-expo`.

Reply:
OK, let's work with the Reanimated team to remove it, maybe.